playbooks/installer/ospd/post_install/scale_swift.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Scaling swift-storage nodes down
+  hosts: undercloud
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  gather_facts: no
+  roles:
+      - {role: "installer/ospd/overcloud/scale/", service: 'swift-storage', amount: 1}
+
+- name: Scaling swift-storage nodes up
+  hosts: undercloud
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  gather_facts: no
+  pre_tasks:
```

Comment:
I'd suggest to add more descriptive comment why we need to wait for 60 minutes here.